Automatic documentation with Sphinx and Read the Docs (#104)
sct-pipeline/csa-atrophy: docs/rescale_dataframe.rst

> Rescale dataframe

Reviewer: ```suggestion
Inter-subject
```

Author: Done

> =====================

Reviewer: ```suggestion
=================
```

> Across-subject statistics are regrouped in the Panda dataframe ``df_rescale``

Reviewer: ```suggestion
Inter-subject statistics. These statistics are gathered in the Panda dataframe ``df_rescale``
```

Author: Implemented in commit c5671c2

> Mean intra-subject SD

Reviewer: ```suggestion
Intra-subject SD averaged across subjects
```
